dlog/plonk-5-wires/src/index.rs

```diff
@@ -82,28 +82,31 @@ where
     G::ScalarField: CommitmentField,
 {
     pub fn verifier_index(&self) -> VerifierIndex<G> {
-        let srs = match &self.srs {
+        let srs0 = match &self.srs {
             SRSValue::Value(s) => SRSValue::Value(s.clone()),
             SRSValue::Ref(x) => SRSValue::Ref(x),
         };
 
+        let trimmed_length = (self.cs.domain.d1.size as u64).trailing_zeros() as usize;
+        let srs = srs0.get_ref().trim(trimmed_length);
```

Comment:
here I'm confused as well. My understand here is that we should trim only when the polynomial we want to commit to fits in `n` such that `n < g.len()` and `n` is a power of 2. So why trim the SRS for all the future commitments here?

Reply:
oh never mind, we only use the trimmed SRS to commit to the pre-computed polynomials, but store the non-trimmed "srs0" in the verifier index.